src/esrs/state.rs

```diff
@@ -25,21 +51,21 @@ impl<S: Default> AggregateState<S> {
     /// might happen.
     ///
     /// Prefer [Default] implementation.
-    #[must_use]
```

Comment:
Removed as this does not mean "you must use AggregateState::new", it means "the return value of AggregateState::new" must be used as a rvalue, e.g.
```
let state = AggregateState::new(id); // <- this is fine
AggregateState::new(id); // <- NOT fine, compiler will warn that the newly created state is not used and dropped immediately
```